specification/e-referrals-service-api.yaml

```diff
@@ -3650,22 +3650,68 @@ paths:
       security:
         - bearerAuth: []
       description: |
-        # TO ADD LATER
-      summary: A041 - Search service request
+        ## Overview
+
+        The e-RS represents an advice and guidance request and a referral in FHIR v4 as a ServiceRequest. This endpoint allows you to search for attributes about a ServiceRequest.
+
+        At present the following is the only permitted use case for using this endpoint, but it is expected that additional use cases will be added as future needs are identified and prioritised.
+        - For FHIR v4 endpoints, the e-RS requires the unique identifier for the ServiceRequest, instead of the UBRN, to identify an advice and guidance request or referral request. To support partners to transition from previous FHIR versions, and utilise the newer FHIR endpoints, this endpoint can be used as a bridging mechanism so that you can search for the required unique identifiers using a UBRN. 
+
+        ## Supported security patterns
+
+        - Healthcare worker, user-restricted access
+
+        ## Pre-requisites
+        To use this endpoint, you must be an authenticated e-RS user and use one of the following e-RS roles
+        - `REFERRING_CLINICIAN`
+        - `REFERRING_CLINICIAN_ADMIN`
+        - `SERVICE_PROVIDER_CLINICIAN`
+        - `SERVICE_PROVIDER_CLINICIAN_ADMIN`
+
+        You need to use the Unique Booking Reference Number (UBRN) of an advice and guidance request or referral request.
+
+        ## Important notes
+
+        When searching for a unique identifier for a ServiceRequest, it’s important to note that one UBRN can relate to both an advice and guidance request and a referral. In this case, the response could contain two unique identifiers: one relating to the advice and guidance request and one relating to the referral.
+
+        ## Use case
+
+        As an authenticated user 
+
+        I need the unique identifier for a given advice request or referral 
+
+        So that I can view/action an advice and guidance request or referral
+
+        ## Related endpoints
+
+        Once the unique identifier has been retrieved it can then be used to call the following endpoint to upload a clinical attachment
+        - [[HYPERLINK_A042]]
+        - [[HYPERLINK_A039]]
+        - [[HYPERLINK_A012]]
+
+        ## Sandbox test scenarios
+        You can test the following scenarios in our sandbox environment
+
+        | Scenario                                                                     | UBRN           |
+        | ---------------------------------------------------------------------------- | -------------- |
+        | A response containing a single Request ID for an Appointment Request         | `000000070000` |
```

Comment:
We should avoid using the "Appointment Request" terminology as this is e-RS specific and not a concept that the outside world will understand. We should just call it a "Referral Request".

Reply:
It should really be named `ServiceRequest` as this is the FHIR resource name.

Reply:
Will change to:

> A response containing a single Request ID for a `ServiceRequest`